crates/mbe/src/mbe_expander/transcriber.rs

```diff
@@ -115,7 +115,9 @@ fn expand_subtree(
     ExpandResult { value: tt::Subtree { delimiter: template.delimiter, token_trees: tts }, err }
 }
 
-fn expand_var(ctx: &mut ExpandCtx, v: &SmolStr) -> ExpandResult<Fragment> {
+fn expand_var(ctx: &mut ExpandCtx, v: Option<&SmolStr>) -> ExpandResult<Fragment> {
+    let underscore = SmolStr::new_inline("_");
+    let v = v.unwrap_or(&underscore);
```

Comment:
This is all awkward because `name: &'a SmolStr` so we can't make a new one when needed.